lib/gooddata/core/logging.rb

```diff
@@ -23,7 +23,7 @@ module GoodData
   DEFAULT_SPLUNKLOG_OUTPUT = STDERR
 
   class << self
-    attr_accessor :logger, :rest_logger, :gd_logger
+    attr_accessor :logger, :rest_logger, :gd_logger, :lcm_logger
```

Comment:
lcm_logger is not a logger. It is just a error message. We should not keep it in this global class.

Reply:
I cannot find any where good enough to store the error message. So could you please suggest me where we can store it ?
Thank you.

Reply:
I think we can define a new custom error (ex: LcmError) that contains all errors. 
In lcm.rb we create this error and throws it to outside. 
In run_brick.rb, we catch the LcmError and handle the error messages

Reply:
Done fix it